theme.inc

```diff
@@ -69,7 +70,10 @@ function _gent_base_get_entity_reference_file_data(&$variables, $field) {
   }
 
   $file_name = $file_entity->getFilename();
-  $file_type = strtoupper(end(explode('.', $file_name)));
+
+  $file_type = explode('.', $file_name);
+  $file_type = end($file_type);
+  $file_type = strtoupper($file_type);
```

Comment:
Moeten we dit in PHP doen of kan dit in CSS?

Reply:
@MPParsley, zoals besproken: we nemen dit op in de algemene refactoring van de `gent_base.theme` file 😉

Reply:
:ok_hand: